tests/storage/cdi_import/conftest.py

```diff
@@ -190,3 +193,37 @@ def vm_list_created_by_multiprocess(dv_list_created_by_multiprocess, storage_cla
     wait_for_processes_exit_successfully(processes=processes, timeout=TIMEOUT_4MIN)
     yield vms_list
     clean_up_multiprocess(processes=processes, object_list=vms_list)
+
+
+@pytest.fixture()
+def dv_and_vms_setup(namespace, storage_class_name_scope_function):
+    dvs = []
+    vms = []
+    for name in ("dv1", "dv2", "dv3"):
+        dv = DataVolume(
+            source=REGISTRY_STR,
+            name=f"import-public-registry-quay-{name}",
+            namespace=namespace.name,
+            url=QUAY_FEDORA_CONTAINER_IMAGE,
+            size=FEDORA_DV_SIZE,
+            storage_class=storage_class_name_scope_function,
+            api_name="storage",
+        )
+        dv.create()
+        dvs.append(dv)
+    for dv in dvs:
+        vm = VirtualMachineForTests(
+            name=dv.name,
+            namespace=namespace.name,
+            os_flavor=OS_FLAVOR_FEDORA,
+            data_volume=dv,
+            memory_requests=FEDORA_VM_MEMORY_SIZE,
+        )
+        vm.deploy()
```

Comment:
Maybe we need to wait for the VM to be created before calling `vm.start()`?
```suggestion
        vm.deploy(wait=True)
```